frontend/package.json

```diff
@@ -19,6 +19,8 @@
     "prop-types": "^15.7.2",
     "react": "^17.0.1",
     "react-card-flip": "^1.0.11",
+    "react-dnd": "^5.0.0",
```

Comment:
Hey @garvitgupta13 do we need these packages? If not, please remove them

Reply:
They are the required dependencies of `react-tags-input`, without them the tag input field will not work.